recipes/marisa/all/conanfile.py

```diff
@@ -0,0 +1,88 @@
+import os
+from conan import ConanFile, tools
+from conan.tools.files import apply_conandata_patches
+from conans import CMake
+
+required_conan_version = ">=1.45.0"
+
+
+class MarisaConan(ConanFile):
+    name = "marisa"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/s-yata/marisa-trie"
+    description = "Matching Algorithm with Recursively Implemented StorAge "
+    license = ("BSD-2-Clause", "LGPL-2.1")
+    topics = ("algorithm", "dictionary", "marisa")
+    exports_sources = "patches/**", "CMakeLists.txt"
```

Comment:
There a more optimal way that we are trying to encourage https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#exporting-patches

Reply:
Yes, I will fix as your suggestion